_includes/scripts/back_to_top.liquid

```diff
@@ -0,0 +1,8 @@
+<script src="{{ '/assets/js/vanilla-back-to-top.min.js' | relative_url | bust_file_cache }}"></script>
+<script>addBackToTop({
+  diameter: 56,
+  cornerOffset: 50, // px
+  backgroundColor: '#ddd',
+  textColor: '#000',
```

Comment:
I believe it would be better if we set some of these values to the ones already set in [_theme.scss](https://github.com/alshedivat/al-folio/blob/master/_sass/_themes.scss) (e.g. `--global-text-color`), or create new ones for these there. This way all styling can be done just by settings css properties.

Reply:
Ok, I added new variables for `back-to-top` button, pls review.